### ISSUE TITLE
download list with vulnerability score

### ISSUE THREAD

**Issue by dalekunce** [home-fire-risk/smoke_alarm_map#26]
It would be really nice to have a way to show/return all the vulnerability scores as a simple csv with census tract numbers within a given Red Cross defined area.

**Comment by dalekunce** [comment]
Another thing to add would be the list in order of vulnerability when you hit print. This could be achieved through hidden divs with @media tags.

**Comment by brooksandrew** [comment]
@hrecht looks like we lost the download to CSV feature with the transition back to single page map?

**Comment by hrecht** [comment]
Weird, definitely not intentional. Probably just got lost in the shuffle. Do either of you have a preference on where to put it @dalekunce @brooksandrew ? Could go on the about section, or the data table section, or a link in the header.

**Comment by dalekunce** [comment]
I would include the overall one in the data table section. I would also allow a user to download only some of the data, Red Cross region, top 50 at risk overall, etc.

**Comment by hrecht** [comment]
@dalekunce @brooksandrew 
Took a while to find the time for this but:
I moved the table creation functions over from the fast but problematic d3 implementation to [DataTables](https://datatables.net/) in a new branch - https://github.com/home-fire-risk/smoke_alarm_map/tree/table-revamp. DataTables gives a much nicer UX  and allows for relatively easy download creation, which I added in https://github.com/home-fire-risk/smoke_alarm_map/commit/810412a87818b4cca1c914f6ffbf12e38089bf8c
The download buttons up top return the filtered data that's in the table (so in this case, just the Chicago and Northern Illinois region data) and also include region name and the full tract geoid. I put the link for the full CSV at the bottom as well, which includes some other columns that we're not displaying and all 73,000ish tracts.

![screen shot 2016-07-09 at 12 49 38 pm](https://cloud.githubusercontent.com/assets/9753760/16709050/a70573a2-45d3-11e6-992c-badb6a363a86.png)

I think the pop-up div for the table was a nice idea but with this extra functionality I found it difficult to use and cluttered, and we would need to also make some performance improvements (Mapbox and DataTables are both pretty heavy). I moved the table to a separate html page, which is linked to in the header. I'm open to exploring it as a pop-up again but think this is a more usable solution.

Thoughts? Issues? I'll open a PR and merge into gh-pages this week if there's no objections.
